lightning/src/ln/functional_test_utils.rs

```diff
@@ -3116,10 +3117,11 @@ pub fn claim_payment_along_route(args: ClaimAlongRouteArgs) {
 	}
 }
 
-pub fn claim_payment<'a, 'b, 'c>(origin_node: &Node<'a, 'b, 'c>, expected_route: &[&Node<'a, 'b, 'c>], our_payment_preimage: PaymentPreimage) {
+pub fn claim_payment<'a, 'b, 'c>(origin_node: &Node<'a, 'b, 'c>, expected_route: &[&Node<'a, 'b, 'c>], our_payment_preimage: PaymentPreimage) -> Option<Bolt12Invoice> {
 	claim_payment_along_route(
 		ClaimAlongRouteArgs::new(origin_node, &[expected_route], our_payment_preimage)
 	);
+	None
```

Comment:
Is this planned to be updated, since it always returns `None` right now?

Reply:
ops! yes thanks for the catch!